prefq/server.py

```diff
@@ -140,27 +134,41 @@ def receive_feedback():
 
     query_id = left_filename[: -len("-left.webm")]
 
-    # Check for defective msg transfer
-    if is_left_preferred is None:
-        return jsonify({"success": False})
+    # Delete videos from queue, if proper feedback is received
+    if is_left_preferred is not None:
```

Comment:
under the current implementation it can, as in [web_interface.js](https://github.com/timokau/prefq/blob/d109f783949404ce2cece3000edbfe5de7963dd5/prefq/static/web_interface.js) the feedback client sends empty boolean values when leaving the page without having given feedback. Under the updated version for issue #29 I already got rid of this behavior